### PR TITLE
[fix] OpenAI model listing

### PR DESCRIPTION
**Pull request by paxcema**
## Description

**[Fixes this issue](https://github.com/mindsdb/mindsdb_private/issues/178)**

## Type of change

(Please delete options that are not relevant)

- [x] 🐛 Bug fix (non-breaking change which fixes an issue)

## Verification Process

To ensure the changes are working as expected:

 - [ ]   Test Location: Specify the URL or path for testing.
 - [ ]   Verification Steps: Outline the steps or queries needed to validate the change. Include any data, configurations, or actions required to reproduce or see the new functionality.

## Additional Media:

- [ ] I have attached a brief loom video or screenshots showcasing the new functionality or change.

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have appropriately commented on my code, especially in complex areas.
- [ ] Necessary documentation updates are either made or tracked in issues.
- [ ] Relevant unit and integration tests are updated or added.

### CODE REVIEW

**Comment by dusvyat** [comment]
lgtm, will def do the job for now

File: mindsdb/integrations/handlers/openai_handler/openai_handler.py [hunk 1]
```diff
@@ -37,7 +37,8 @@ def __init__(self, *args, **kwargs):
         self.max_batch_size = 20
         self.default_max_tokens = 100
         self.chat_completion_models = CHAT_MODELS
-        self.supported_ft_models = ('davinci', 'curie', 'babbage', 'ada')  # base models compatible with finetuning
+        self.supported_ft_models = FINETUNING_LEGACY_MODELS  # base models compatible with finetuning  # TODO #7387: transition to new endpoint before 4/1/24 # noqa
+        self.engine_storage.json_set('ft-suffix', binascii.b2a_hex(os.urandom(15)))  # user suffix for finetunes
```
Comment (paxcema) [inline mindsdb/integrations/handlers/openai_handler/openai_handler.py:41]: Missing a `.decode()` here to cast into string.

**Comment by paxcema** [comment]
Hey @dusvyat if there's nothing else, could you formally approve to unblock merging? Thanks!

**Comment by paxcema** [comment]
@ea-rus tagging you for review on that last commit -- apparently the json setter/getter for our engine storage was not implemented. Please have a quick look 🙏

**Review by ea-rus — APPROVED** [review]

**Review by dusvyat — APPROVED** [review]

**Pull request merged**